akka-actor/src/main/java/akka/dispatch/forkjoin/package-info.java

```diff
@@ -4,7 +4,6 @@
  * http://creativecommons.org/publicdomain/zero/1.0/
  */
```

Comment:
Good that it doesn't change that

Reply:
Do we have any other headers that have both Lightbend and some other copyright notice? When we copy code from elsewhere (when the license permits) we preserve the original copyright notice.

Reply:
akka-protobuf

Reply:
akka-persistence